runtime/src/main/kotlin/kotlin/native/internal/test/TestStatistics.kt

```diff
@@ -26,26 +26,44 @@ internal class MutableTestStatistics: TestStatistics {
     override var totalSuites: Int = 0; private set
 
     override val failed: Int
-        get() = failedTests_.size
+        get() = _failedTests.size
```

Comment:
Why _ before? Generally _ after gives better tooling hints.

Reply:
According to the coding conventions: https://kotlinlang.org/docs/reference/coding-conventions.html#property-names.

Reply:
Hmm, maybe coding convention better be rethought here. Anyway, not in this PR.